src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colour colour;
```

Comment:
Please ensure that you are using the correct naming convention for constants. The `Colour` enum should be in all uppercase letters if it is intended to be used as a constant.

Reply:
Fixed